docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,284 @@
+Z11_SPF1_SYNTAX_ERROR           | ERROR   | ns_ip_list, spf_policy        | The following SPF version 1 policy has a syntax error: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF1_SYNTAX_OK              | INFO    |                               | The *Child Zone*’s SPF version 1 policy has correct syntax.
+Z11_SPF1_TOO_COMPLEX            | ERROR   | ns_ip_list, spf_policy, count | Evaluating the following SPF version 1 policy would require {count} recursive DNS lookups by an SPF validator, exceeding the limit of 10: {spf_policy}. Policy retrieved from the following nameservers: {ns_ip_list}.
+Z11_SPF_WRONG_VERSION           | WARNING | ns_ip_list, spf_version       | The *Child Zone* publishes an SPF policy with invalid version number {spf_version}. Policy retrieved from the following name servers: {ns_ip_list}.
+Z11_UNABLE_TO_CHECK_SPF         | ERROR   |                               | Unable to check whether *Child Zone* publishes an SPF policy.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+## Test procedure
+
+In this section and unless otherwise specified below, the term "[DNS Query]"
+follows the specification for DNS queries as specified in [DNS Query and
+Response Defaults]. The handling of the DNS responses on the DNS queries follow,
+unless otherwise specified below, what is specified for [DNS Response] in the
+same specification.
+
+1. Create a [DNS Query] with query type TXT and query name *Child Zone* ("TXT
+   query").
+
+2. Create two empty sets of strings, "SPF-Policies" and "SPFv1-Policies".
+
+3. Obtain the set of name server IP addresses using [Method4] and [Method5]
+   ("Name Server IP").
+
+4. For each name server in *Name Server IP* do:
+
+   1. Send *TXT Query* to the name server and collect the [DNS Response].
+
+   2. Go to the next name server IP address if at least one of the following
+      criteria is met:
+
+      1. There is no DNS response.
+      2. [RCODE Name] of the response is not "NoError".
+      3. The AA flag is not set in the response.
+      4. There is no TXT resource record in the response.
+
+   3. If the name server responds with at least one TXT record, then, for each
+      TXT record:
+
+      1. If the record is an [SPF TXT record], then add the lowercased
+         [concatenation][concatenate] of all strings in the TXT record’s data to
+         the *SPF-Policies* set.
+
+   4. Go to the next name server.
+
+5. Compare the set of *SPF-Policies* retrieved from all name servers. If at
+   least one name server has responded with a set of SPF policies and a second
```

Comment:
I think "with a set" --> "with one set" is more idiomatic.

Reply:
I’m going to rephrase this sentence anyway because it’s awkward. I’m considering:

> If at least two different name servers have returned different sets of SPF policies, then output *[Z11_INCONSISTENT_SPF_POLICIES]*.